src-built-in/components/toolbar/src/app.jsx

```diff
@@ -6,14 +6,21 @@
 // Static vs Dynamic Toolbar
 import Toolbar from "./dynamicToolbar";
 // import Toolbar from "./staticToolbar";
-//Band-aid. Openfin not respecting small bounds on startup.
-if (!fin.container || fin.container != "electron") {
-  fin.desktop.main(() => {
-    let finWindow = fin.desktop.Window.getCurrent();
+
+function FSBLReady() {
+  //Band-aid. Openfin not respecting small bounds on startup.
+  if (!FSBL.System.container || FSBL.System.container != "electron") {
+    let finWindow = FSBL.System.Window.getCurrent();
```

Comment:
I know we are removing references to `fin` in 4.0, but should we also modify names like `finWindow` or is that okay as it is?

Reply:
That's fine. I'll follow up with my rationale.